Parsers/ASimWebSession/Parsers/vimWebSessionApacheHTTPServer.yaml

```diff
@@ -0,0 +1,113 @@
+Parser:
+  Title: Web Session ASIM parser for Apache HTTP Server
+  Version: '0.1'
+  LastUpdated: Jun 19, 2023
+Product:
+  Name: Apache HTTP Server
+Normalization:
+  Schema: WebSession
+  Version: '0.2.6'
+References:
+- Title: ASIM Web Session Schema
+  Link: https://aka.ms/ASimWebSessionDoc
+- Title: ASIM
+  Link: https:/aka.ms/AboutASIM
+- Title: Apache HTTP Server 
+  Link: https://httpd.apache.org/docs/2.4/logs.html
+Description: |
+  This ASIM parser supports filtering and normalizing Apache HTTP Server logs produced by the Microsoft Sentinel Apache connector to the ASIM Web Session normalized schema.
+ParserName: ASimWebSessionApacheHTTPServer
+EquivalentBuiltInParser: _ASim_NetworkSession_ApacheHTTPServer
+ParserParams:
+  - Name: disabled
+    Type: bool
+    Default: false
+ParserQuery: |
+  let parser=(
+    starttime:datetime               = datetime(null), 
+    endtime:datetime                 = datetime(null),
+    srcipaddr_has_any_prefix:dynamic = dynamic([]),
+    ipaddr_has_any_prefix:dynamic    = dynamic([]), 
+    url_has_any:dynamic              = dynamic([]),
+    httpuseragent_has_any:dynamic    = dynamic([]),
+    eventresultdetails_in:dynamic    = dynamic([]),
+    eventresult:string               = '*',
+    disabled:bool                    = false
+  ){
+    let src_or_any                = set_union(
+      srcipaddr_has_any_prefix,
+      ipaddr_has_any_prefix
+    ); 
+    let remove_protocol_from_list = (list:dynamic)
+    {
+        print list
+        | mv-apply l = print_0 to typeof(string) on
+          ( extend l = replace_regex (tostring(l), "^(?i:.*?)://", "") )
```

Comment:
can we avoid regex? with something like substring(l,indexof(l,@'//')+2)

Reply:
I like the logic, thanks!